modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMqContainer.java

```diff
@@ -0,0 +1,107 @@
+package org.testcontainers.containers;
+
+import java.time.Duration;
+
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+
+
+/**
+ * This {@link Container} is based on the official RabbitMQ ({@code rabbitmq}) images from
+ * <a href="https://hub.docker.com/_/rabbitmq/">DockerHub</a>.
+ *
+ * @author Stefan Ludwig
+ */
+public class RabbitMqContainer extends GenericContainer<RabbitMqContainer> {
+
+    private static final String DEFAULT_LOG_MESSAGE_REGEX = ".*Server startup complete.*\n";
+    private static final int DEFAULT_STARTUP_TIMEOUT = 30;
+
+    /**
+     * This is the internal port on which RabbitMQ is running inside the container.
+     * <p>
+     * You can use this constant in case you want to map an explicit public port to it
+     * instead of the default random port. This can be done using methods like
+     * {@link #setPortBindings(java.util.List)}.
+     */
+    public static final int RABBITMQ_PORT = 5672;
+
+    /**
+     * Creates a new {@link RabbitMqContainer} with the {@code 'rabbitmq:3.7'} image.
+     */
+    public RabbitMqContainer() {
+        this("3.7");
+    }
+
+    /**
+     * Creates a new {@link RabbitMqContainer} with the {@code 'rabbitmq:${tag}'} image.
+     * <p>
+     * You can use the tag {@code 'latest'} in order to always execute your tests against
+     * the newest available version. But it is generally recommended to match the version
+     * of your target environment.
+     *
+     * @param tag the tag (e.g. version) to use
+     */
+    public RabbitMqContainer(@NotNull String tag) {
```

Comment:
could you please make it accept an image instead? Some users under corporate policies reported that they want to change the images (to something like `my.corp.org:1234/cool/rabbitmq-image`) and current implementation doesn't allow that

Reply:
ok, I'll do that ASAP. Hopefully I'll have some time tomorrow.